src/components/transcript/transcript.tsx

```diff
@@ -224,16 +238,29 @@ export class Transcript extends Component<TranscriptProps, TranscriptState> {
           kitchenSinkActive={kitchenSinkActive}
         />
         <TranscriptMenu {...{downloadDisabled, onDownload, printDisabled, onPrint, isLoading}} />
-        <div data-testid="transcriptCloseButton">
+        <div data-testid="transcriptDetachAttachButton">
           <Button
             type={ButtonType.borderless}
             size={ButtonSize.medium}
-            disabled={false}
-            onClick={this.props.onClose}
-            ariaLabel={'Hide Transcript'}
-            tooltip={{label: 'Hide Transcript'}}
-            icon={'close'}></Button>
+            onClick={kitchenSinkDetached ? onAttach : onDetach}
+            icon={kitchenSinkDetached ? 'attach' : 'detach'}
+            ariaLabel={kitchenSinkDetached ? 'Bring Transcript back' : 'Popout Transcript'}
```

Comment:
why is the text hardcoded instead of using i18n ?

Reply:
resolved